src/olympia/hero/serializers.py

```diff
@@ -33,6 +34,15 @@ def to_representation(self, instance):
         rep.pop('addon' if instance.is_external else 'external')
         return rep
 
+    def get_description(self, obj):
+        if obj.description:
+            return ugettext(obj.description)
+        else:
+            addon = obj.disco_addon.addon
+            return (str(addon.summary)
```

Comment:
Where does this requirement come from? Was that baked into `serializers.CharField(source='disco_addon.description_text')`?

Reply:
yeah, check `DiscoveryItem.description_text`.